features/user.feature

```diff
@@ -0,0 +1,90 @@
+Feature: User Registration
+  Creating a User
+
+  Scenario: I want to create a user with all required parameters
+    When  To create user I send a POST request to "api/users/create" with the following
```

Comment:
```suggestion
    When  I send a POST request to "api/users/create" with the following
```

Reply:
When  I send a POST request to "api/users/create" with the following

If I use this i am getting conflict with other features. That is why I have added - To create user

Reply:
What was the reason of conflict? Have you checked that?

Reply:
I have used same text in booking feature. while running tests I am getting "ambiguous match" error.